lib/manager/flux/index.ts

```diff
@@ -1,8 +1,36 @@
 import { HelmDatasource } from '../../datasource/helm';
+import { logger } from '../../logger';
+import { exec } from '../../util/exec';
+import type { ExecOptions } from '../../util/exec/types';
+//import { doAutoReplace } from '../../workers/branch/auto-replace';
+import type { UpdateDependencyConfig } from '../types';
 export { extractAllPackageFiles, extractPackageFile } from './extract';
 
 export const defaultConfig = {
   fileMatch: [],
 };
 
 export const supportedDatasources = [HelmDatasource.id];
+
+export async function updateDependency({
+  upgrade,
+}: UpdateDependencyConfig): Promise<string> {
+  if (upgrade.packageFile.endsWith('/flux-system/gotk-components.yaml')) {
+    logger.debug(`Updating flux-system manifests`);
+    const cmd = 'flux install --export';
+    const execOptions: ExecOptions = {
+      docker: {
+        image: 'fluxcd/flux-cli',
```

Comment:
We need to extend our buildpack to dynamically install to `sidecar` docker image and to support install mode.

Reply:
Yeah, I figured something like that would be necessary - where do I configure that? Maybe it was obvious but I missed it.

Reply:
Needs to be done at https://github.com/containerbase/buildpack. You can open a corresponding issue there.

Reply:
https://github.com/containerbase/buildpack/pull/250